src/Components/FAQ.js

```diff
@@ -62,118 +62,225 @@ const FAQ = props => {
               </p>
+            </Collapsible>
+          </SingleItem>
+        </Gradient>
+
+        <Gradient>
+          <SingleItem>
+            <Collapsible
+              trigger="Are asset allocations on the Pie Network binary options, derivatives, CFDs, futures, or options?"
+              easing="ease-in"
+              transitionTime={100}
+              triggerTagName="div"
+            >
               <p>
-                It can even be another Collapsible component. Check out the next
-                section!
+                Asset allocations on Pie Network are created by stakeholder of the Pie Dao and deployed by individual users of the Pie smart contracts.
               </p>
             </Collapsible>
           </SingleItem>
         </Gradient>
+
         <Gradient>
           <SingleItem>
             <Collapsible
-              trigger="Is it better than Compound?"
+              trigger="What does the Pie Dao do and what does it not do?"
               easing="ease-in"
               transitionTime={100}
               triggerTagName="div"
             >
               <p>
-                This is the collapsible content. It can be any element or React
-                component you like.
+                The Pie Dao writes and publishes open source software on Github. There is a test website for developers at dev.pie.network running on the test network of Ethereum (play money). The Pie Network is a set of Solidity smart contracts that exist on the Ethereum blockchain.
               </p>
               <p>
-                It can even be another Collapsible component. Check out the next
-                section!
+                Among other things, the Pie Dao does not:
+                <ul>
+                  <li>Operate an exchange</li>
+                  <li>Create markets or contracts on the Pie Network</li>
+                  <li>Escrow, custody, transfer, or otherwise handle funds or cryptocurrencies on the Pie Network</li>
+                  <li>Control who accesses the Pie Network</li>
+                </ul>
               </p>
             </Collapsible>
           </SingleItem>
         </Gradient>
+
         <Gradient>
           <SingleItem>
             <Collapsible
-              trigger="Is it safe?"
+              trigger="Which protocol are used inside the Pie Network smart contracts?"
               easing="ease-in"
               transitionTime={100}
               triggerTagName="div"
             >
               <p>
-                This is the collapsible content. It can be any element or React
-                component you like.
+              Pie Dao uses existing DeFi protocols as building blocks to unlock the compounding value of multiple financial primitives. Pie Network combines the use of synthetic asset platforms, and decentralized exchanges. 
```

Comment:
Maybe use "The Pie Dao" consistently. With "the" and consistent capitalisation of Dao vs DAO